locations/spiders/bashas.py

```diff
@@ -0,0 +1,56 @@
+import html
+import json
+import scrapy
+
+from locations.dict_parser import DictParser
+from locations.hours import OpeningHours
+
+
+class BashasSpider(scrapy.spiders.SitemapSpider):
+    name = "bashas"
+    item_attributes = {
+        "brand": "Bashas'",
+        "brand_wikidata": "Q4866786",
+    }
+
+    download_delay = 0.2
+    sitemap_urls = ["https://www.bashas.com/wpsl_stores-sitemap.xml"]
+    sitemap_rules = [("/stores/", "parse_store")]
+
+    def parse_store(self, response):
+        script = response.xpath(
+            "//script[@id='wpsl-js-js-extra']/text()"
+        ).extract_first()
+        open_brace = script.find('"locations":[{') + 13
+        close_brace = script.rfind("}]}") + 1
+        store_json = json.loads(script[open_brace:close_brace])
+        hours = response.xpath("//table[@class='wpsl-opening-hours']//text()").extract()
+
+        item = DictParser.parse(store_json)
```

Comment:
It looks like the DictParser is picking up an address and setting it as `addr_full` when it should be `street_address`. Can you add a `item["street_address"] = item.pop("addr_full", None)` somewhere after this line?

Reply:
I made this change. I noticed the same problem was happening with a spider I added yesterday (Gelsons), so I made the change there too.